### PR TITLE
Zone activation guide for OIDC sudoers

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] Try to isolate changes into separate PRs (to build a better changelog).
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `change`, `decision`, `requirement/quality`, `requirement/functional`, `dependency`
      as they show up in the changelog

### CODE REVIEW

**Review by glrf — APPROVED** [review]

File: docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ yq -i eval ".parameters.openshift4_terraform.terraform_variables.cluster_name =
 .For cloudscale.ch
 [source,bash]
 ----
-yq -i eval ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"plus-32-4\"" \
+yq -i eval ".parameters.openshift4_terraform.terraform_variables.infra_flavor = \"plus-24\"" \
```
Comment (simu) [inline docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc:72]: This seems unrelated, and IMO `plus-32-4` is the correct infra node flavor.

```suggestion
```

**Review by simu — CHANGES_REQUESTED** [review]

File: docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc (hunk 1 above)
Comment (bastjan) [reply docs/modules/ROOT/pages/how-to/zone-cluster-install.adoc:72]: Yes it is 🙈 Seemed to happen in thee rebase. Weird 😅
